src/main/webapp/i18n/de/exam.json

```diff
@@ -310,38 +310,73 @@
                 "registerAllFromCourseTooltip": "Registriere alle zu diesem Kurs angemeldeten Studierende",
                 "removeFromExam": {
                     "modalQuestion": "Soll die Person mit dem Login <strong>{{ title }}</strong> wirklich von dieser Prüfung abgemeldet werden? Ihre individuelle Klausur wird dabei gelöscht.",
-                    "deleteParticipationsAndSubmission": "Teilnahme und Abgaben in der zugehörigen Klausur löschen?"=======
                     "deleteParticipationsAndSubmission": "Zusätzlich die Teilnahme und Abgaben in der zugehörigen Klausur löschen? Diese Aktion kann NICHT rückgängig gemacht werden!",
                     "typeNameToConfirm": "Bitte gib den Login der Person zur Bestätigung ein."
-                },
-
                 }
+            }
```

Comment:
Not all changes seem to be reflected in the English version.
Is this a fix or did you forget it?

Reply:
This was a fix, i moved all the translations into a new file called importUsers.json

Reply:
I'm not referring to this change. Please compare it to /en/exam.json

There you do two things: Remove "importStudents" to place it in another file and move "studentExamDetail" one layer up.
But in /de/exam.json you do do additional stuff in the lines between (exerciseGroup(s), delete, checklist ...)

Reply:
Ah okay. Due to a merge conflict, i had a couple of duplicated lines... so i had to compare all of them by hand and removed the duplicates